docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -42,6 +42,8 @@ Malware protection levels are:
 +
 TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
 
+Malware protection also allows you to enable a blocklist of specified applications that are prevented from running in host environments, extending the list of processes that Endpoint Security considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To configure specific applications on the blocklist, refer to <<blocklist>>.
```

Comment:
```suggestion
Malware protection also allows you to enable a blocklist of specified applications that are prevented from running in host environments, extending the list of processes that Endpoint Security considers malicious. Use the **Blocklist enabled** toggle to enable or disable this feature for all hosts associated with the integration policy. To manage applications on the blocklist, refer to <<blocklist>>.
```

Reply:
Going with a more concise version of the final sentence (in comment above), as the reference doesn't need to be that specific.